fireant/database/data_blending.py

```diff
@@ -0,0 +1,29 @@
+from .type_engine import make_columns
+
+
+class DataBlender:
+    def __init__(self, primary_db, primary_type_engine, secondary_db, secondary_type_engine):
+        self.primary_db = primary_db
+        self.primary_type_engine = primary_type_engine
+        self.secondary_db = secondary_db
+        self.secondary_type_engine = secondary_type_engine
+
+    def derive_columns_from_secondary(self, secondary_schema, secondary_table):
```

Comment:
what's the difference between this method and the next? what's the workflow of using this class?